common/changes/@uifabric/webpack-utils/loader_2018-06-06-20-44.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/webpack-utils",
+      "comment": "Creates a new webpack-utils package",
+      "type": "major"
```

Comment:
Your first version published is going to be 2.0.0...

Reply:
yeah maybe update the version to a 0, and do minor bumps until you're ready to do 1.0

Reply:
ok

Reply:
I cranked it down to 0.0.1 with a minor change file